### PR TITLE
add home page button

### PR DESCRIPTION
**Pull request by allie-rae**
Screen Shot 2022-08-18 at 4 13 32 PM
Screen Shot 2022-08-18 at 4 13 46 PM

### CODE REVIEW

**Review by danimal-johnson — APPROVED** [review]
Short. Clear. To the point. Excellent.

File: src/pages/Layout/Layout.tsx [hunk 1]
```diff
@@ -38,7 +39,7 @@ const Layout = (props: any) => {
 
   const drawer = (
     <Box onClick={handleDrawerToggle} sx={{ textAlign: 'center' }}>
-      <Typography variant="h6" sx={{ my: 2 }} onClick={() => navigate('/')}>
+      <Typography variant="h6" sx={{ my: 2, cursor: pointer }} onClick={() => navigate('/')}>
```
Comment (allie-rae) [inline src/pages/Layout/Layout.tsx:42]:
```suggestion
      <Typography variant="h6" sx={{ my: 2, cursor: 'pointer' }} onClick={() => navigate('/')}>
```

File: src/pages/Layout/Layout.tsx [hunk 2]
```diff
@@ -38,7 +39,7 @@ const Layout = (props: any) => {
 
   const drawer = (
     <Box onClick={handleDrawerToggle} sx={{ textAlign: 'center' }}>
-      <Typography variant="h6" sx={{ my: 2 }} onClick={() => navigate('/')}>
+      <Typography variant="h6" sx={{ my: 2, cursor: 'pointer' }} onClick={() => navigate('/')}>
```
Comment (allie-rae) [inline src/pages/Layout/Layout.tsx:42]:
```suggestion
      <Box sx={{ cursor: 'pointer' }}>
          <Typography variant="h6" sx={{ my: 2 }} onClick={() => navigate('/')}>
      </Box>
```

**Review by jae-finger — APPROVED** [review]
Good to go!

**Review by squashgray — APPROVED** [review]

**Pull request merged**